content/sql/concepts/primary-keys/primary-keys.md

```diff
@@ -12,12 +12,12 @@ CatalogContent:
   - 'paths/analyze-data-with-sql'
 ---
 
-SQL tables sometimes have a column that uniquely identifies each row of that table. These special columns are called primary keys.
+**Primary keys** are special columns that are used to uniquely identify each row of a table in SQL.
 
 A primary key column has a few requirements:
 
 - None of the values can be `NULL`.
-- Each value must be unique (i.e., you can’t have two customers with the same `customer_id` in the `customers` table).
+- Each value must be unique (e.g., two rows in a `customers` table wouldn't have the same primary `customer_id`).
 - A table can not have more than one primary key column.
```

Comment:
Is this actually the case given the new information about composite keys?

Reply:
Hi Dusch! I believe the statement is accurate given the example we're studying there. Mentioning composite keys or how a field or column could appear with the same value in different rows at this moment is—in my opinion—something that would confuse the reader. Besides the concept itself, composite keys are known for being not the best design, performance-wise. I think the doc is super clear as it is now!

Reply:
Gotcha! Any thoughts, @SSwiniarski?

Reply:
@Dusch4593 I think we should delete the word "column" so it just says "A table cannot have more than one primary key."

Reply:
```suggestion
- A table can not have more than one primary key.
```